sstables/generation_type.hh

```diff
@@ -22,6 +22,7 @@
 #include <seastar/core/sstring.hh>
 #include "types/types.hh"
 #include "utils/UUID_gen.hh"
+#include "utils/UUID_cmp.hh"
```

Comment:
I think you made practically the entire source base include them again.  I don't know if the patch makes any difference.